js/abp-preprocessed.es6.js

```diff
@@ -4,64 +4,72 @@
  *
  * This will be browserifyed and turned into abp.js by running 'grunt'
  */
-abp = require('abp-filter-parser');
+abp = require('abp-filter-parser')
 const ONEDAY = 1000*60*60*24
 
-easylists = {
-    privacy: {
-        url: 'https://duckduckgo.com/contentblocking.js?l=easyprivacy',
-        parsed: {},
+let lists = {
+    easylists : {
+        privacy: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=easyprivacy',
+            parsed: {},
+        },
+        general: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=easylist',
+            parsed: {},
+        }
     },
-    general: {
-        url: 'https://duckduckgo.com/contentblocking.js?l=easylist',
-        parsed: {},
+    whitelists: {
+        preWhitelist: {
+            url: 'https://duckduckgo.com/contentblocking.js?l=trackers-whitelist',
+            parsed: {}
```

Comment:
Part of the reason for this code being open source is so that we are transparent to the user about what we are doing. Naming this `preWhitelist` doesn't give the user any insight into the what/why here. Can we call this `ddgWhitelist`  and leave a comment with link to the open source repo that contains the list?